aqt/updater.py

```diff
@@ -274,8 +272,8 @@ def update(cls, target: TargetConfig, base_dir: str):
                 arch_dir = b + "_" + a
             else:
                 arch_dir = arch[6:]
-        elif version in SimpleSpec(">=6.1.2") and os_name == "mac" and arch == "clang_64":
-            arch_dir = "macos"
+        elif os_name == "mac" and target.target == "desktop":
+            arch_dir = default_desktop_arch_dir(os_name, version)
```

Comment:
This change is inappropriate because it breaks the macos/wasm_32 build. Perhaps this aggressive refactoring was a mistake.